packages/storybook/cxl-ui/cxl-app-layout/layout=1c-hp.stories.js

```diff
@@ -0,0 +1,55 @@
+import { html } from 'lit';
+import { withKnobs, boolean, text } from '@storybook/addon-knobs';
```

Comment:
All new code should be Controls, not Knobs (deprecated).

https://github.com/storybookjs/storybook/blob/next/code/addons/controls/README.md#how-do-i-migrate-from-addon-knobs

Our SB is partially Controls, example https://github.com/conversionxl/aybolit/blob/4f66ea97711e60504abd51b8ca69dd56ca15e8dd/packages/storybook/cxl-lumo-styles/elements.stories.js#L19